packages/storybook/src/example-pages/Patronen/zoeken-in-lijst-cards.ts

```diff
@@ -0,0 +1,124 @@
+import { html } from "lit-html";
+
+import { examplePageFactory } from "../../example-page-factory";
+import { footerPartial } from "../partials/footer";
+import { headerPartial } from "../partials/header";
+import { header } from "../partials/header.content";
+import { cardList } from "./zoeken-in-lijst-cards.content";
+
+examplePageFactory(
+  "Patronen",
+  "Zoeken in lijst cards",
+  (
+    {
+      applicationHeadingTemplate,
+      searchBarTemplate,
+      badgeTemplate,
+      cardListTemplate,
+      selectableTemplate,
+      footnoteTemplate,
+    },
+    templates
+  ) => html`
+    <div class="container">
+      {% render '@header', {menu: menu, breadcrumbs: breadcrumbs} %}
```

Comment:
Dit zal nog wel een WIPje zijn, maar in ieder geval moeten we deze niet vergeten

Reply:
Zeker een wip, zal hem niet uit het oog verliezen